test/deriver/errors/default_str_no_manifest.ml

```diff
@@ -0,0 +1 @@
+type t [@@deriving default]
```

Comment:
Minor: Should this be on two lines so it looks more like our code?

Reply:
Yep I'll change that!